### DISCUSSION TITLE
React Native - Coinbase Smart Wallet support

### DISCUSSION THREAD

**Discussion by emzet93** [WalletConnect/walletconnect-monorepo#5356] [Developer Support]
Is it possible to connect to coinbase smart wallet using `@web3modal/wagmi-react-native`?
Coinbase Smart Wallets is available only as the browser app so far, using passkeys for accessing wallets. I imagine integration would have to go through mobile browser somehow.

**Comment by arein** [comment]
Coinbase Smart Wallet isn't available for native dapps yet. Once it is we will support it

**Reply by emzet93** [reply]
I think it is now: https://www.smartwallet.dev/guides/react-native-integration

Integration is actually pretty similar to what you currently have with standard Coinbase Wallet App https://docs.walletconnect.com/appkit/react-native/core/installation#enable-coinbase-wallet
It also works great with wagmi.

I'm attaching a short demo from our app using Coinbase Smart Wallet with slightly modified version of `@web3modal/wagmi-react-native`.

https://github.com/user-attachments/assets/3b6a4881-4e1b-4781-88c0-cec9f8d9ee69

I would be happy to work on or help with PR integrating that directly in @web3modal/wagmi-react-native when I find some free time. Let me know if it's something you would be interested in. Thanks!

**Comment by ANN-BLAIR** [comment]
Realizing I was a victim of a scam was a devastating blow. My initial investment of $89,000, driven by dreams of financial success and the buzz surrounding a new cryptocurrency project, turned into a nightmare. The project promised high returns and rapid gains, attracting many eager investors like myself. However, as time passed and inconsistencies began to surface, it became evident that I had made a grave mistake by not thoroughly vetting the brokerage company handling the investment. Feeling anxious and betrayed, I desperately searched for a way to recover my funds. During this frantic search, I stumbled upon the Brigadia Tech Recovery tool through a Facebook post. With little left to lose, I reached out to their team for help. To my relief, they were quick to respond and immediately started recovering my compromised email and regaining access to my cryptocurrency wallets. The team at Brigadia Tech Recovery was incredibly professional and transparent throughout the process. They meticulously traced the digital footprints left by the scammers, employing advanced technological methods to unravel the complex network that had ensnared my funds. Their expertise in cybersecurity and recovery strategies gradually began to turn the tide in my favor. Although the scammers had already siphoned off $30,000 worth of Bitcoin, Brigadia Tech Recovery was relentless in their pursuit. They managed to expose the fraudulent activities of the scam operators, revealing their identities and the mechanisms they used to lure investors. This exposure was crucial not only for my case but also as a warning to the wider community about the perils of unverified investment schemes. As we progressed, it became a race against time to retrieve the remaining $59,000 before the scammers could vanish completely. Each step forward was met with new challenges, as these criminals constantly shifted tactics and moved their digital assets to evade capture. Nonetheless, the determination and skill of the recovery team kept us hopeful. Throughout this ordeal, I learned the hard value of caution and due diligence in investment, especially within the volatile world of cryptocurrency. The experience has been incredibly taxing, both emotionally and financially, but the support and results provided by Brigadia Tech Recovery have been indispensable. Currently, the recovery process is ongoing and I have already received 50% of my money, and while the outcome remains uncertain, the progress made so far gives me hope. The battle to recover the full amount of my investment continues, and with the expertise of Brigadia Tech Recovery, I remain optimistic about the eventual recovery of my funds. Their commitment to their clients and proficiency in handling such complex cases truly sets them apart in the field of cyber recovery.I share this testimony because i am sure someone out there is going through the same issue and devastated mentally by all these fake brokers all around the world,This is the main contact information incase you are interested to recover your lost investments 
Email:(Brigadiatechremikeable (@) Proton.Me)
﻿Telegram +1 (323) 910-1605)
